Exercise_B/index.html

```diff
@@ -19,16 +24,119 @@
 
   <body>
     <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
+    <nav>
+      <img src="assets/images/logo.png">
+      <!--1st widget-->
+      <button type="button" class="btn btn-link"><a href="#info">BUY NOW</a></button>
+    </nav>
+
+    <header>
+      <div><h1>Beautiful Flowers</h1></div>
+      <div class="break"></div>
+      <div class="button"><a href="#info"><h3>Learn more</h3></a></div>
+    </header>
 
     <!--Four containers with img / text-->
+    <section id="info">
     <!-- 4 containers -->

```

Comment:
The page is somewhat responsive. I am happy that you were testing how to override bootstrap code nice job!

However, we are  asking you to use the bootstrap column system to put all of the content (icons and descriptions) onto  one line on an extra large screen
2 icons and their text per line on a medium screen
1 icon and its content per line on a small screen

Come back and add in another class on your div containers

<div>
<img ...
<p>....
</div>